.github/release-drafter.yml

```diff
@@ -0,0 +1,5 @@
+# see https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.adoc
+_extends: .github
+version-template: $MAJOR.$MINOR.$PATCH
```

Comment:
This plugin still uses full semver and not major.minor as other Jenkins plugins.